packages/ses/test/test-compartment-make-scope-proxy.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-underscore-dangle */
+import test from 'ava';
+import '../index.js';
+
+lockdown();
+
+test('compartment main use case exploded', t => {
+  function power(a) {
+    return a + 1;
+  }
+
+  const c1 = new Compartment({ power });
+  const { scopeProxy: scopeProxy1 } = c1.__makeScopeProxy__();
+  const power1 = scopeProxy1.power;
+  const attenuatedPower = arg => {
+    if (arg <= 0) {
+      throw new TypeError('only positive numbers');
+    }
+    return power1(arg);
+  };
+
+  const c2 = new Compartment({ power: attenuatedPower });
+  const { scopeProxy: scopeProxy2 } = c2.__makeScopeProxy__();
```

Comment:
```suggestion
  const { scopeProxy: scopeProxy2 } = __makeScopeProxy__({ power: attenuatedPower });
```

My intuition remains that this function is independent of `Compartment` and that `Compartment` should not be used at all if you’re wiring scope proxies directly.

Reply:
my reason to center this around the compartments because it represents a small change in the code and mental model around evaluating modules in the context of compartments

```js
// what we want but cannot have
compartment.evaluate(moduleSrc)
// something similar that avoids eval
magicWrappedFn.call(compartment.__makeScopeProxy__())
compartment.__runMagicWrappedFn__(magicWrappedFn)
```